integrations/oci/oci-secrets-mp-config-source/src/main/java/io/helidon/integrations/oci/secrets/mp/configsource/OciSecretsMpMetaConfigProvider.java

```diff
@@ -93,4 +94,9 @@ public Set<String> supportedTypes() {
         return this.p.supported();
     }
 
+    @Override
+    public ConfigSource create(Reader content) {
```

Comment:
This belongs to the default method implementation, this provider should not be modified at all.